.github/workflows/release-ecr.yml

```diff
@@ -0,0 +1,53 @@
+name: Release Apps for ECR
+
+on:
+  release:
+    types: [ published ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  APPS: 'legacy-api,ocean-api,playground-api,status-api'
+
+jobs:
+  build:
+    if: github.actor != 'dependabot[bot]'
+    name: Publish
+    runs-on: ubuntu-latest
+    environment: ECR Release Publishing
+    strategy:
+      matrix:
+        app: [ legacy-api, ocean-api, playground-api, status-api ]
```

Comment:
I am unsure should this includes `rich-list-api`. What do you think?

Reply:
Regarding `rich-list`, since its currently on hold right now and not completed (see https://github.com/JellyfishSDK/jellyfish/pull/1173#issuecomment-1110768560), I think we can exclude it for now and add it later.